### ISSUE TITLE
Can you tell me more detail about training on UCF-101

### ISSUE THREAD

**Issue by imnotk** [yoosan/i3d-tensorflow#3]
I train I3d model on UCF-101 split 1 with pretrained Kinetics-imagenet weights, but I only can get the accuracy on test split about 45% , but train accuracy about 95% on rgb modality.
So how did you get the 95.2% accuracy on split 1  , can you give me some advise?
